integration/docker/csi/alluxio/controllerserver.go

```diff
@@ -122,7 +126,6 @@ func (cs *controllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 		glog.V(3).Infof("Invalid delete volume req: %v", req)
 		return nil, err
 	}
-	glog.V(4).Infof("Deleting volume %s", volumeID)
```

Comment:
We don't do anything here. The log is misleading.

Reply:
Does the returned value not trigger a deletion? If so why is the return type not a `CreateVolumeResponse`?
`return &csi.DeleteVolumeResponse{}, nil`

Reply:
Actually the returned value will trigger the deletion of the pv. However that is not happening inside this function, so logging should also not be here. Plus we are not removing any data stored in Alluxio